### ISSUE TITLE
[Bug]: In app preview never loads

### ISSUE THREAD

**Issue by DastardlyDino** [astubenbord/paperless-mobile#357]
### What happened?

When trying to view scanned images true in app preview never loads.

### Steps to reproduce

1. Open app
2. Login
3. Click on document
4. Click upper small image preview to expand
5. Full image preview never loads

![Screenshot_20240110_053125](https://github.com/astubenbord/paperless-mobile/assets/47339617/8a10ffa9-ca84-46b3-abc0-c786cdbbc856)

### Log output from the app

```shell
2024-01-10 05:22:02.119	DEBUG   --- [      AuthenticationCubit] - restoreSession           : Trying to restore previous session...
2024-01-10 05:22:02.119	DEBUG   --- [      AuthenticationCubit] - restoreSession           : There is nothing to restore.
2024-01-10 05:22:02.121	INFO    --- [                         ] -                          : Setting refresh rate to 120.00000762939453
2024-01-10 05:22:40.561	DEBUG   --- [      AuthenticationCubit] - login                    : Trying to log in Andrew@pa***yz...
2024-01-10 05:22:40.562	DEBUG   --- [      AuthenticationCubit] - _addUser                 : Adding new user Andrew@pa***yz..
2024-01-10 05:22:40.562	DEBUG   --- [      AuthenticationCubit] - _addUser                 : Fetching bearer token from the server...
2024-01-10 05:22:40.948	DEBUG   --- [      AuthenticationCubit] - _addUser                 : Bearer token successfully retrieved.
2024-01-10 05:22:40.949	DEBUG   --- [      AuthenticationCubit] - _getApiVersion           : Trying to fetch API version...
2024-01-10 05:22:40.978	WARNING --- [      AuthenticationCubit] - _getApiVersion           : The server is running a newer API version (4) than the app supports (v3), falling back to latest supported version (v3). Warning: This might lead to unexpected behavior!
2024-01-10 05:22:40.978	DEBUG   --- [      AuthenticationCubit] - _getApiVersion           : Successfully retrieved API version (3).
2024-01-10 05:22:40.978	DEBUG   --- [      AuthenticationCubit] - _addUser                 : Trying to fetch remote paperless user for Andrew@pa***yz.
2024-01-10 05:22:41.065	DEBUG   --- [      AuthenticationCubit] - _addUser                 : Remote paperless user successfully fetched.
2024-01-10 05:22:41.065	DEBUG   --- [      AuthenticationCubit] - _addUser                 : Persisting user account information...
2024-01-10 05:22:41.073	DEBUG   --- [      AuthenticationCubit] - _addUser                 : User account information successfully persisted.
2024-01-10 05:22:41.073	DEBUG   --- [      AuthenticationCubit] - _addUser                 : Persisting user app state...
2024-01-10 05:22:41.075	DEBUG   --- [      AuthenticationCubit] - _addUser                 : User state successfully persisted.
2024-01-10 05:22:41.386	DEBUG   --- [      AuthenticationCubit] - _addUser                 : Saving user credentials inside encrypted storage...
2024-01-10 05:22:41.387	DEBUG   --- [      AuthenticationCubit] - _addUser                 : User credentials successfully saved.
2024-01-10 05:22:41.388	DEBUG   --- [      AuthenticationCubit] - _addUser                 : Added new url to list of hosts.
2024-01-10 05:22:41.392	DEBUG   --- [      AuthenticationCubit] - login                    : User Andrew@pa***yz successfully logged in.
2024-01-10 05:22:41.399	DEBUG   --- [          LabelRepository] - findAllTags              : Loading all tags...
2024-01-10 05:22:41.399	INFO    --- [               InboxCubit] - refreshItemsInInboxCount : Checking for new documents in inbox...
2024-01-10 05:22:41.431	DEBUG   --- [          LabelRepository] - findAllTags              : Loading all tags...
2024-01-10 05:22:41.432	INFO    --- [               InboxCubit] - refreshItemsInInboxCount : Checking for new documents in inbox...
2024-01-10 05:22:41.649	INFO    --- [               InboxCubit] - refreshItemsInInboxCount : No new documents found in inbox.
2024-01-10 05:22:41.649	DEBUG   --- [          LabelRepository] - findAllTags              : Loading all tags...
2024-01-10 05:22:41.705	DEBUG   --- [          LabelRepository] - findAllTags              : Successfully updated all tags.
2024-01-10 05:22:41.706	DEBUG   --- [          LabelRepository] - findAllTags              : Successfully updated all tags.
2024-01-10 05:22:41.716	INFO    --- [               InboxCubit] - refreshItemsInInboxCount : No new documents found in inbox.
2024-01-10 05:22:41.716	DEBUG   --- [          LabelRepository] - findAllTags              : Loading all tags...
2024-01-10 05:22:41.748	DEBUG   --- [          LabelRepository] - findAllTags              : Successfully updated all tags.
2024-01-10 05:22:41.772	DEBUG   --- [          LabelRepository] - findAllTags              : Successfully updated all tags.
2024-01-10 05:23:11.438	INFO    --- [               InboxCubit] - refreshItemsInInboxCount : Checking for new documents in inbox...
2024-01-10 05:23:11.565	INFO    --- [               InboxCubit] - refreshItemsInInboxCount : No new documents found in inbox.
2024-01-10 05:23:41.445	INFO    --- [               InboxCubit] - refreshItemsInInboxCount : Checking for new documents in inbox...
2024-01-10 05:23:41.568	INFO    --- [               InboxCubit] - refreshItemsInInboxCount : No new documents found in inbox.
2024-01-10 05:24:11.434	INFO    --- [               InboxCubit] - refreshItemsInInboxCount : Checking for new documents in inbox...
2024-01-10 05:24:11.570	INFO    --- [               InboxCubit] - refreshItemsInInboxCount : No new documents found in inbox.
2024-01-10 05:24:41.444	INFO    --- [               InboxCubit] - refreshItemsInInboxCount : Checking for new documents in inbox...
2024-01-10 05:24:41.644	INFO    --- [               InboxCubit] - refreshItemsInInboxCount : No new documents found in inbox.
2024-01-10 05:26:56.070	INFO    --- [               InboxCubit] - refreshItemsInInboxCount : Checking for new documents in inbox...
2024-01-10 05:26:56.236	INFO    --- [               InboxCubit] - refreshItemsInInboxCount : No new documents found in inbox.
```

### Screenshots

![Screenshot_20240110_053125](https://github.com/astubenbord/paperless-mobile/assets/47339617/38ce111a-95ab-4b7e-895e-d9eaef37defb)

### Paperless-ng*x Version

2.3.2

### Paperless-Mobile Version

3.2.0+404

### Device

s23 ultra

### Operating System Version

android 14 

### Device Locale

USA 

### Selected Locale

English (US)

**Comment by Dtrieb** [comment]
Same here.

**Comment by hermannoffen** [comment]
This issue should be closed being another duplicate of #350.

**Comment by astubenbord** [comment]
Yeah, I think this issue has already been reported 5 times already. This was fixed but the new packages does not seem to compile in the runner's environment (only god knows why...). I'll see what I can do today.

Closing as duplicate of #350.
